packages/core/admin/ee/server/services/__tests__/audit-logs.test.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-const { features } = require('@strapi/strapi/lib/utils/ee');
+const { features } = require('@strapi/strapi/dist/utils/ee');
 const { register } = require('@strapi/provider-audit-logs-local');
 const { scheduleJob } = require('node-schedule');
 const createAuditLogsService = require('../audit-logs');
-const createEventHub = require('../../../../../strapi/lib/services/event-hub');
+const createEventHub = require('../../../../../strapi/dist/services/event-hub').default;
```

Comment:
is there a reason we can't do `const { createEventHub } = require('@strapi/strapi')`? Also why is it still being produced as an esm module? 😬

Reply:
I'm not making refactorings in this PR if I can avoid them let's not make this more difficult that it needs to be for reviewers.

Reply:
Okay even if we don't change the path the fact you have to add `.default` feels off to me, this could be caused by `@strapi/strapi` now being an ESM module by default which iirc is a breaking change.

Reply:
This is transpiled to commonjs or it wouldn't even be loadable. the problem is the default behavior of TS compilation. 

https://github.com/microsoft/TypeScript/issues/2719

Reply:
So if we're using the tsc bundling method this might have the same issue as we're having with `@strapi/utils` where the types aren't properly accessible? Have we tested that the current level of typescript support still actively works with this method of bundling 🤔

Reply:
you mean export = ? a huge chunk of the node ecosystem uses this method to be backward compatible while providing typings

Reply:
> a huge chunk of the node ecosystem uses this method to be backward compatible while providing typings

Sure, but we were having issues with `@strapi/utils` so before we merge this we should probably test some TS projects 😄

Reply:
Yes I had do create a js file to make it compat and have the typings but they are not really accurate because of that.